packages/node_modules/@webex/plugin-meetings/src/common/events/util.js

```diff
@@ -6,10 +8,10 @@ EventsUtil.getEventLog = (args) => {
   let argString = '';
 
   try {
-    argString = JSON.stringify(args);
+    argString = inspect(args);
   }
   catch (e) {
-    LoggerProxy.logger.info(`Events:util#getEventLog --> ${e}`);
+    LoggerProxy.logger.warn(`Events:util#getEventLog --> ${e}`);
```

Comment:
I wasn't sure what this was actually for. There doesn't seem to be any info as to why we would print this out or what a user could do. I guess warn is fine, but what are we warning?

Reply:
We are warning that there is a bug in our code. The user would need to submit an issue with the message in it.

Reply:
Sounds good!